docs/_static/css/custom.css

```diff
@@ -7,3 +7,7 @@ textarea{
   border: 0;
   padding: 0;
 }
+
+article p {
```

Comment:
better doc formatting?

Reply:
yea, the docs had no spaces between paragraphs and it drove me crazy haha Much more readable now